### ISSUE TITLE
Suppress certain error types

### ISSUE THREAD

**Issue by sfkeller** [kort/kort#47]
Sometimes users want to suppress certain error types, because they are only interested in certain ones. 

Cases where one dislikes certain error types are 1. in areas where a single error type is predominant or 2. where there's no answer (like the language of the name tag in multiligual areas, see "name=Biel/Bienne" or Bozen http://osm.org/go/0IQBA_UBB-- , http://666kb.com/i/cf2mh64tyakrc3td2.jpg , where in fact several name:XX should be inserted).

So in the users profile there could be a select/filter dialog, to display only selected error types (saved in users preferences).

**Comment by harry-wood** [comment]
Yeah I'm seeing a lot of 'language of the name'  in my area (UK) which I'm not convinced is useful data to be adding really. I could be wrong about that, but...   the point is I'd like to see less of that error type, 

That could be either through an explicit option to switch them off, or...

maybe it should just be smarter about showing less of the same type of question. Maybe it's already doing something smart about that (?)  but this would also be helped by having more different question types.
